release.md

```diff
@@ -0,0 +1,612 @@
+
+### esda:
+* enh: updating travis build and rtd [(#40)](https://github.com/pysal/esda/pull/40)
+* BUG: missing rtd file [(#39)](https://github.com/pysal/esda/pull/39)
+* REL: 2.0.1 [(#38)](https://github.com/pysal/esda/pull/38)
+* Prepping for a doc release [(#37)](https://github.com/pysal/esda/pull/37)
+* docstrings are using pysal legacy [(#4)](https://github.com/pysal/esda/issues/4)
+* add zenodo doi badge [(#36)](https://github.com/pysal/esda/pull/36)
+* REL: 2.0.0 [(#34)](https://github.com/pysal/esda/pull/34)
+* Changing esda setup to handle version programatically [(#33)](https://github.com/pysal/esda/pull/33)
+* port legacy esda fix for 1013  [(#12)](https://github.com/pysal/esda/issues/12)
+* notebook links broken [(#29)](https://github.com/pysal/esda/issues/29)
+* include /tests in release [(#32)](https://github.com/pysal/esda/pull/32)
+* Add tests to release [(#27)](https://github.com/pysal/esda/issues/27)
+* Accounting for incoming API changes to `libpysal` and adding testing against `libpysal`'s master branch [(#26)](https://github.com/pysal/esda/pull/26)
+* no `varnames` in `Moran_BV_matrix` [(#22)](https://github.com/pysal/esda/issues/22)
+* add `.varnames` attribute to `Moran_BV` objects in `Moran_BV_Matrix` results [(#23)](https://github.com/pysal/esda/pull/23)
+* Inconsistent metadata in setup.py [(#28)](https://github.com/pysal/esda/issues/28)
+* Update license [(#30)](https://github.com/pysal/esda/pull/30)
+* esda has no readme [(#14)](https://github.com/pysal/esda/issues/14)
+* Readme added [(#25)](https://github.com/pysal/esda/pull/25)
+* move api into __init__ and remove api [(#24)](https://github.com/pysal/esda/pull/24)
+* `.z` attribute divided by standard deviation [(#21)](https://github.com/pysal/esda/pull/21)
+* offer standardised and non-standardised `.z`, `.x` and `.y` attributes [(#20)](https://github.com/pysal/esda/issues/20)
+* get sjsrey's changes into a release [(#16)](https://github.com/pysal/esda/pull/16)
+* esda's namespace is broken [(#17)](https://github.com/pysal/esda/issues/17)
+* update api.py [(#18)](https://github.com/pysal/esda/pull/18)
+* Update docstrings to use libpysal not pysal [(#13)](https://github.com/pysal/esda/pull/13)
+* esda needs an api.py module [(#9)](https://github.com/pysal/esda/issues/9)
+* chore: Update setup for 3+ [(#15)](https://github.com/pysal/esda/pull/15)
+* Master [(#10)](https://github.com/pysal/esda/pull/10)
+
+### giddy:
+* remove giddy.api in README.rst [(#66)](https://github.com/pysal/giddy/pull/66)
+*  chore: update for libpysal lower case module name changes [(#65)](https://github.com/pysal/giddy/pull/65)
+* remove api.py [(#62)](https://github.com/pysal/giddy/issues/62)
+* set up travis dual testing against mapclassify and  esda [(#63)](https://github.com/pysal/giddy/issues/63)
+* replace `libpysal.api` imports with new imports in `markov.py` and `d… [(#61)](https://github.com/pysal/giddy/pull/61)
```

Comment:
seems to have an extra "`" here which is causing formatting to get out of whack.

Reply:
Yup. But I think still  [renders ok](https://github.com/sjsrey/pysal/blob/next/release.md)